lib/segment/tests/integration/byte_storage_quantization_test.rs

```diff
@@ -46,40 +48,68 @@ enum QuantizationVariant {
     Binary,
 }
 
-fn random_discovery_query<R: Rng + ?Sized>(rnd: &mut R, dim: usize) -> QueryVector {
+fn random_vector<R>(rnd_gen: &mut R, dim: usize, data_type: VectorStorageDatatype) -> DenseVector
```

Comment:
because we test binary quantization, we need specify vector to utilise binary condition. for `u8` we generate vectors in range `[0; 255]`, for `f16` we generate vectors in range `[-0.5; 0.5]`